evap/staff/views.py

```diff
@@ -2263,6 +2265,23 @@ def faq_section(request, section_id):
     return render(request, "staff_faq_section.html", template_data)
 
 
+@manager_required
+def infotexts(request):
+    InfoTextFormSet = modelformset_factory(Infotext, form=InfotextForm, extra=0)
+    formset = InfoTextFormSet(request.POST or None, queryset=Infotext.objects.all())
```

Comment:
codecov says this is not covered -- can we have a simple "can be opened, stuff can be entered, can be submitted, and stuff in the DB is then changed" test?